brhstaging.data-commons.org/manifest.json

```diff
@@ -20,7 +20,7 @@
     "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2023.03",
     "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2023.03",
     "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2023.03",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2023.03",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:5.5.0",
```

Comment:
Can't we have a single pull hey request where we update both portal and hatchery?

Reply:
Yeah, that's the idea. Hatchery PR (https://github.com/uc-cdis/hatchery/pull/61) was failing jenkins, I rebuilt a few times, its rebuilding again now. Hopefully that's a success and we can cut a release and add it here.